Resources/Prototypes/tags.yml

```
@@ -221,8 +221,10 @@
   id: BulletFoam
 
 - type: Tag
-  id: BypassInteractionRangeChecks
+  id: BypassDropChecks
 
+- type: Tag
+  id: BypassInteractionRangeChecks
 - type: Tag
```

```suggestion

- type: Tag
  id: BypassInteractionRangeChecks
  
- type: Tag
```

I'm gonna slap github